Dockerfile

```diff
@@ -38,6 +38,16 @@ RUN mkdir -p /code/unstable_languages
 COPY views /code/views
 COPY webtranslate /code/webtranslate
 
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    \
+    make \
+    && rm -rf /var/lib/apt/lists/*
+COPY docs /docs
+RUN make -C /docs && \
+    mv /docs/manual/_build/html /code/static/docs && \
+    rm -r /docs
+
```

Comment:
Ideally, we do a multi stage Dockerfile so we don't get `make` in the final image.

Check website's Dockerfile if you wonder how that works.